Sources_v3/Utils/Internet Connection/InternetConnection.swift

```diff
@@ -146,12 +156,11 @@ private extension InternetConnection {
         func stop() {
             monitor?.cancel()
             monitor = nil
-            delegate?.internetConnectionStatusDidChange(status: .unknown)
         }
         
         private func createMonitor() -> NWPathMonitor {
             let monitor = NWPathMonitor()
-            monitor.pathUpdateHandler = { [unowned self] in self.updateStatus(with: $0) }
+            monitor.pathUpdateHandler = { [weak self] in self?.updateStatus(with: $0) }
```

Comment:
Why? The original version seems to be more correct.

Reply:
It crashed on this point. Easy to reproduce, but difficult to say why.